src/main/kotlin/com/github/quiltservertools/ledger/actions/ItemDropActionType.kt

```diff
@@ -23,7 +22,7 @@ open class ItemDropActionType : AbstractActionType() {
 
     override fun getTranslationType(): String {
         val item = Registries.ITEM.get(objectIdentifier)
-        return if (item is BlockItem && item !is AliasedBlockItem) {
+        return if (item is BlockItem) {
```

Comment:
This breaks with items like wheat seeds, which are block items but only have item translations